services/api/scripts/jobs/example.js

```diff
@@ -1,9 +1,10 @@
 const process = require('process');
-const { logger } = require('@bedrockio/instrumentation');
+const logger = require('@bedrockio/logger');
 const { initialize } = require('../../src/utils/database');
 const { sleep } = require('../../src/utils/sleep');
 
 async function run() {
+  logger.setupGoogleCloud();
```

Comment:
no if production flag?

Reply:
The idea here is that jobs are only ever run by yacron in production. In contrast scripts are run manually in the CLI.
I could put a check here but there isn't really ever a need and I want to reduce the jobs boilerplate as you'll be copying these.